pkg/test/resource_creator_test.go

```diff
@@ -58,20 +59,21 @@ func TestGetOperatorNamespace(t *testing.T) {
 			t.Errorf("expected %v, got %v", operatorNamespace, got)
 		}
 	})
-	t.Run("should return non-empty new Operator Namespace, when OperatorNamespace Env = \"\"", func(t *testing.T) {
-		operatorNamespace := ""
-		Global.OperatorNamespace = operatorNamespace
-		defer func() {
-			Global.OperatorNamespace = ""
-		}()
-		ctx := NewContext(t)
-		//defer os.Unsetenv(OperatorNamespaceEnv)
-		got, err := ctx.GetOperatorNamespace()
-		assertNoError(t, err)
-		if len(got) <= 0 {
-			t.Errorf("expected non-empty operatorNamespace")
-		}
-	})
+	t.Run("should return non-empty new Operator Namespace, when OperatorNamespace Env = \"\"",
+		func(t *testing.T) {
```

Comment:
Jus change the indentation 👍

Reply:
@camilamacedo86 the func is intented like this to preserve the 80 character lint rule.
i have kept the intendation consistent in other test cases.

Reply:
the indendation is automatically done by `go fmt`